C/c4CAPI.cc

```diff
@@ -10,6 +10,7 @@
 // the file licenses/APL2.txt.
 //
 
+#include "c4Base.h"
```

Comment:
That's not necessary. That header is transitively included by the other API headers (e.g. c4BlobStore.hh), and none of the other headers (to my knowledge) redundantly include them.

Reply:
removed.